pkgs/servers/http/apache-modules/mod_auth_mellon/fixdeps.patch

```diff
@@ -1,8 +1,23 @@
+diff --git a/Makefile.in b/Makefile.in
```

Comment:
Has there been any attempts to upstream this patch?

Reply:
Not that I'm aware of. Since `.ac` files are black magic to me I'm probably not in a position to do so either.

Reply:
@jtojnar I'll make a PR upstream and hope they understand the black magic enough to merge it 😄 

Thanks for **always** holding the highest of standards.